### DISCUSSION TITLE
Feature Request: Animation/Visual indication when moving workspaces

### DISCUSSION THREAD

**Discussion by GallowsDove** [YaLTeR/niri#657] [Ideas]
I'd really appreciate an animation when moving workspaces (e.g. Super + Shift + PgUp/PgDown). 
Currently it doesn't indicate that you moved a workspace at all, and I find myself accidentaly moving workspaces without even realizing and then being confused where my windows disappeared.

Alternatively there should at least be some indication that something happened.

By the way, thanks for your work on Niri! It's pretty amazing so far, and can't wait to see where this will be in the future.

**Comment by YaLTeR** [comment]
I agree some animation would be nice, but I'm not sure what it could look like since the workspace covers the whole screen.

As a workaround, you should soon be able to use a workspaces module on Waybar (currently with a git niri build and my Waybar PR).

> By the way, thanks for your work on Niri! It's pretty amazing so far, and can't wait to see where this will be in the future.

Thanks!
